src/components/jobManager.ts

```diff
@@ -346,6 +346,8 @@ export class JobManager {
 			this.#jobMap.delete(jobHash);
 		}
 
+		messages.push({ kind: MessageKind.updateElements });
```

Comment:
was it accidentally removed?

Reply:
no, this line was missing from the beginning, just checked the git timeline